lib/Checks/NoteCheck.vala

```diff
@@ -37,19 +37,14 @@ public class ValaLint.Checks.NoteCheck : Check {
                     if (index > 0) {
                         /* Get message of note */
                         int index_newline = r.text.index_of ("\n", index);
-                        int index_end = int.min (r.text.length, index_newline);
+                        int index_end = (index_newline > -1) ? int.min (r.text.length, index_newline) : r.text.length;
                         string message = r.text.slice (index + keyword.length + 1, index_end).strip ();
 
-                        /* Get correct position of note */
-                        int line_count = Utils.get_line_count (r.text[0:index]);
-                        int line = r.begin.line + line_count;
-                        int column = Utils.get_column_in_line (r.text, index);
-                        if (line_count == 0) {
-                            column += r.begin.column;
-                        }
+                        print (@"$index_end\n");
```

Comment:
This looks like a debugging message that should have been removed?

Reply:
Oh sure, my bad 🙈